frontend/dockerfile/instructions/parse.go

```diff
@@ -209,7 +209,7 @@ func parseKvps(args []string, cmdName string, location []parser.Range, lint *lin
 			return nil, errBlankCommandNames(cmdName)
 		}
 		name, value, sep := args[j], args[j+1], args[j+2]
-		if sep == "" {
+		if lint != nil && sep == "" {
 			msg := linter.RuleLegacyKeyValueFormat.Format(cmdName)
 			lint.Run(&linter.RuleLegacyKeyValueFormat, location, msg)
```

Comment:
Maybe safer to do this check in the `lint.Run()` method, but that can be future follow-up.

Reply:
We should probably find a way to make it so calling linter methods is always valid. I think this is good to just take care of the current issue though.